setup.cfg

```diff
@@ -0,0 +1,22 @@
+[metadata]
+# replace with your username:
+name = luna-featurevis
+version = 0.1
+author = Alex Bäuerle
+author_email = alex.baeuerle@uni-ulm.de
+description = Feature Visualization with Tensorflow 2
+long_description = file: README.md
+long_description_content_type = text/markdown
+url = https://github.com/Sparkier/luna
+classifiers =
+    Programming Language :: Python :: 3
+    License :: OSI Approved :: MIT License
+    Operating System :: OS Independent
+
+[options]
+packages = find:
+python_requires = >=3.6
+install_requires =
+    tensorflow
+    tf_slim
+    keras
```

Comment:
Is numpy not needed? (image.py uses it)

Reply:
I think numpy is a dependency of TensorFlow, which is why it still works I assume. However, if we use it directly we should add it. I will do that.